src/Servers/Kestrel/Transport.Quic/src/PublicAPI.Shipped.txt

```diff
@@ -1,24 +1,24 @@
 #nullable enable
 Microsoft.AspNetCore.Hosting.WebHostBuilderMsQuicExtensions
-Microsoft.AspNetCore.Server.Kestrel.Transport.Experimental.Quic.QuicTransportOptions
```

Comment:
Do not change this file. Instead copy the lines you deleted into PublicAPI.Unshipped.txt with a `*Removed*` prefix

Reply:
This change is why the CodeCheck job failed